language-design.md

```diff
@@ -101,8 +201,30 @@ you choose to include natural language in the design of a DSL? If so, how might
 you do so? If not, why not?
 
 **Response**
-
-
+We believe that there should be a balance in designing programming languages
+between natural language and structured technical terms.
+
+The reason we cannot go to a full natural language is because there are so many
+grammars to the same language. For instance, in English, one stereotype is that
+a person from the South says "Y'all" while a person from the North says "You guys."
+Both words mean the same thing, and there are millions of variations of this,
+especially for a language as widespread as English.
+Additionally, one word can mean different things in different contexts. For
+example, in England, a rubber is an eraser while, in American slang, a rubber
+is a condom. This leads to confusion between humans who speak the same language,
+so it is extremely difficult to expect an artificial machine to understand the nuances.
+
+On the other hand, Pavlus' article explains the problems with too many
+technical terms. He mentions the steep learning curve involved with Perl and
+how it creates a very selective community of those can program [Pavlus, 2012].
+
+In the design of a DSL, we would want structured natural language, similar to
+what we were talking about in the sound lab. In that example, we chose to move
+to object oriented programming because it is closer to reading natural language.
+It reads as a noun (the object) doing verbs in order (the first method listed
+is the first one that is called). However, it must still have the shape
+of object.method(). This way, method calls fall into a certain pattern, while
+still allowing for them to read like a sentence.
```

Comment:
I really enjoy your argument. Designing a programming language with some natural language feeling is a balancing act and requires considerations of all implications of including natural language.